packages/orchestration/src/util/filtering.test.ts

```diff
@@ -21,16 +21,16 @@ describe('Content filter util', () => {
     });
 
     it('builds filter config with no config', async () => {
-      const filterConfig = buildAzureContentSafetyFilter();
+      const filterConfig = buildAzureContentSafetyFilter('input');
```

Comment:
[pp] Consider documenting this usage in documentation portal. Do you also have a PR for documentation open ?